pywbem/cim_operations.py

```diff
@@ -4326,14 +4326,21 @@ def IterEnumerateInstances(self, ClassName, namespace=None,
             instances, as follows:
 
             * If `False`, inherited properties are not excluded.
-            * If `True`, inherited properties are basically excluded, but the
+            * If `True`, the behavior depends: If pull operations are used,
```

Comment:
Suggest change "depends" to "depends on underlying operation"

Reply:
Done